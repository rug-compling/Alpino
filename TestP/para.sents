Ik ben door de buurvrouw naar huis gebracht .
Ik was door de buurvrouw naar huis gebracht .
Ik word door de buurvrouw naar huis gebracht .
Ik werd door de buurvrouw naar huis gebracht .
De kinderen die door de buurvrouw werden verzorgd werden door de buurvrouw verzorgd .
De kinderen die door de buurvrouw zijn verzorgd zijn door de buurvrouw verzorgd .
Wie waren toen door de buurvrouw verzorgd ?
Wie werden toen door de buurvrouw verzorgd ?
Een bijzonderheid : de enquête werd door het eigen personeel van Fina uitgevoerd .
Een tweehonderdtal exemplaren van het nieuwe gereedschap wordt weer gedurende een langere periode getest door professionele gebruikers .
Dat systeem werd ontwikkeld door het TNO in Delft op vraag van grote gebouweigenaars en van de schoonmaakindustrie .
Elke nieuwe werf wordt opgestart door onze meest ervaren ambulante objectleider .
Musea die nog niet aan de eisen kunnen voldoen , worden door museumconsulenten begeleid .
De ondertussen ontwikkelde manuals worden momenteel bestudeerd door diverse CEN-comités .
Ook het eindproduct zelf , bijvoorbeeld een autonavigatie-cd , wordt door Quality Control getest .
Deze meestal dure systemen worden in eerste instantie warm onthaald door gebruikers
Die wordt er door de Papi-methodiek uitgehaald .
Nochtans wordt vandaag de dag de oplossing hiervoor , nl. een gezond verliesbeheersingsbeleid , door slechts enkele bedrijven in België toegepast .
Dit wordt dan weer gecompenseerd door de uitstraling van deze Lancia .
Het Brits-Duitse Rover ( Rover werd door BMW overgenomen ) is een apart merk .
Doordringen op een fleetmarkt , die beheerst wordt door de Europese merken , was voor Mitsubishi niet evident .
Denk maar aan de omniumverzekering op voertuigen die door grote gespecialiseerde maatschappijen worden geleast .
De wagens worden zo door de concessiehouders in de handel gebracht .
Deze specificaties sluiten merken die misschien via een andere techniek eenzelfde doelstelling bereiken dikwijls uit .
Want door de aanwezigheid van een justitie-antenne wordt een wijk meteen gestigmatiseerd .
Toch werd me dat door het kabinet gevraagd .
Aan de Albertsquare zijn de samenlevingsproblemen door de gemeente in een twijfelachtig kunstwerk gegoten .
Drievierde van de flats wordt door Belgen bewoond -- een uitzondering in deze omgeving .
Het Belgisch voetbal is van een gewisse dood gered door Franky Van der Elst .
In de Uefabeker was Moeskroen eerder al kansloos weggeveegd door Metz .
Toch wordt deze bundel gered door drie uitstekende stukken .
Dat werd bevestigd door uitgebreid onderzoek .
Magic wordt wereldwijd door meer dan drie miljoen ( geregistreerde ) spelers beoefend .
Die worden na hun ontslag opgevangen door het outplacementbureau New Challenge Company .
Hij werd opgevoed door zijn oom Proxenus .
Dit wordt door opdrachtgevers vaak als keurmerk gezien .
Zij worden in Andorra vertegenwoordigd door een afgevaardigde .
Sommige gebieden worden bedreigd door gevaren met een menselijke oorzaak .
In 2003 werd het noorden door een zware aardbeving getroffen .
De eerste minister wordt benoemd door de president .
Larry Wall werd door AWK geïnspireerd bij het ontwerpen van de programmeertaal Perl .
Volgens sommige auteurs is aids gemaakt door mensen .
In haar verdere leven werd ze geplaagd door ziekten .
In 1636 werd Aruba overgenomen door de Nederlanders .
Aswoensdag wordt voorafgegaan door vastenavond en het carnaval
Het verschil tussen armoede en welstand wordt aangeduid door de armoedegrens .
Het amok maken in Maleisië is eind 17e eeuw beschreven door Reynier Adriaensen .
Incourante(re) afstanden waarvoor door de IAAF records worden bijgehouden zijn de volgende .
Het antisemitisme is niet door Hitler uitgevonden .
Hij was vergezeld door de minister van Defensie en Ontwikkelingssamenwerking van Luxemburg .
Er vloeide veel geld naar de speciale regelingen die door de regering waren opgelegd .
Maar de premier is opgejut door Karel De Gucht .
Hij weet dat hij door zeer velen wordt gehaat .
Nutsbedrijven ' oude stijl ' werden genadeloos door de beursgigant opgeslokt .
Sindsdien werd hij altijd vergezeld door bodyguards .
Computers worden immers door mensen bediend .
De markt wordt meer en meer gemaakt door de pensioenfondsen .
De negatieve koersevolutie werd door de meeste fondsen beperkter gehouden .
Die verandering is op haar beurt veroorzaakt door een markttrend .
Het beheer van het netwerk wordt trouwens binnenshuis verzorgd door IBM Global Network .
Vooral op lange termijn wordt door een goede voorbereiding veel tijd gewonnen .
Dit remsysteem wordt door velen beschouwd als het veiligste ter wereld .
Het bedrijf was ondertussen door een Japanse groep overgenomen .
Ik ben gevraagd door een Westvlaamse kartonfabriek die in Uganda wil investeren .
De bouw van de stand werd overigens door Mazda-importeur Beherman zelf ter hand genomen .
Waarom moest Aziza onderduiken voor neonazi's en werd haar nicht door neofascisten belaagd ?
De ziekte van Alzheimer wordt veroorzaakt door diverse genetische fouten .
Onze nationale chocolade-olifant werd in 1987 door de Zwitsers gekaapt .
Het plunderen wordt hier blijkbaar in de hand gewerkt door het toerisme .
Wij wensen niets liever dan dat ons werk door onbevooroordeelde specialisten wordt beoordeeld .
Zo wandel ik op straat en word ik door niemand herkend .
Iedereen wordt gedreven door seks , geweld en hebzucht .
Gezonde lucht wordt verpest door rokers .
Door uw cijfers wordt die suggestie wel gewekt .
De Ambonees werd door de Japanse bezetter gewantrouwd .
Alleen wat zich gemakkelijk aandient wordt door media opgepakt .
De surfer wordt door Tony ' Scarface ' Mantana voor fucking cockroach uitgemaakt . '
Lange tijd is ook de geschiedschrijving in Latijns Amerika door deze gedachte beïnvloed .
Planten maken gifstoffen om te voorkomen dat ze door dieren worden opgegeten .
De modderstroom is veroorzaakt door hevige regens .
Bomaanslagen worden zelden gepleegd door gewone mensen .
Zo is internationaal onderzocht welke vrouwen door mannen attractief worden gevonden .
Die stoffen worden dus naar alle waarschijnlijkheid door het insect aangeleverd .
ik word daar wel door uitgedaagd
ik word er wel door uitgedaagd
ik word hier wel door uitgedaagd
ik word overal door uitgedaagd
ik word nergens meer door uitgedaagd
ik word ergens door uitgedaagd
hierdoor word ik uitgedaagd
daardoor word ik uitgedaagd
ik word erdoor uitgedaagd
Door sommigen is toen geprotesteerd
Door sommigen werd toen geprotesteerd
Aan sommige tafeltjes wordt door oudere dames geapplaudiseerd .
Er werd nu eenmaal zwak gespeeld door vrijwel alle spelers .
Buiten werd daaraan herinnerd door tientallen demonstranten .
En daar wordt door sommigen meer gekocht dan normaal .
Tegen die beslissing is vannacht heftig geprotesteerd door pro-Russische betogers .
Buiten het gerechtshof werd betoogd door slachtoffers van de apartheid .
Er wordt door niemand aan getwijfeld wie de winnaar zal zijn van de verkiezingen .
Ook in Italië wordt druk geoefend door grondtroepen .
Welk virus het nu precies is wordt onderzocht door het Nationaal Influenza-centrum in Rotterdam .
Enkele duizenden van hen worden nauwelijks bereikt door hulporganisaties .
Tot de betoging was opgeroepen door de linkse vakbond .
Al is door de aanstelling van de nieuwe partijleider Faymann waarschijnlijk erger voorkomen .
Een peperduur toestel waar dag en nacht op geoefend wordt door leerling-piloten .
In Iran is opnieuw massaal betoogd door de tegenstanders van president Ahmadinejad .
Wanneer Boonen opnieuw zal koersen , wordt door Quick-Step ten gepaste tijde meegedeeld .
Over de crisis wordt straks nog overlegd door het Midden-Oosten kwartet .
Tot de optocht is opgeroepen door de familie .
Vermoedelijk is daar achter de schermen voor geijverd door de Verenigde Staten .
Gisteren was daar al toe opgeroepen door het parlement .
Het is niet de eerste keer dat er gestaakt wordt door het Franse overheidspersoneel .
Op zijn aftreden wordt al een tijdje aangedrongen door de aandeelhouders van Fortis .
Daarom wordt er al massaal gehamsterd door de Spanjaarden .
Ook nu werd er geschoten door de politie .
Dat er een akkoord is , is door de electriciteitsproducenten nog niet bevestigd .
Mogelijk is er gevuurd door een lokale militie van Zuid-Ossetië .
Tot de betoging was opgeroepen door de linkse vakbond CGTP .
Veel van zijn werken zijn door de Arabische geleerde Averroes vertaald .
ik had de schat gevonden , maar hij werd bewaakt door een enorm zeemonster .
Ik werd geteisterd door te veel dingen tegelijk .
De gordijnen waren open en haar hologige gezicht werd omlijst door het vroege ochtendlicht .
Plotseling werd die stilte ruw verstoord door een schorre vloek en een plons .
Toen pa hem volgde , werd hij van achteren beslopen door twee motorrijders .
Het merendeel van de ruimte werd in beslag genomen door een bekraste keukentafel .
' Ik geloof dat je werd geholpen door mijn moeder .
Hij werd gegrepen door het soort boosheid dat zich als verbazing voordoet .
Het begint erop te lijken dat de brand is gesticht door een buitenstaander . '
Werd die immers niet gesteund door Opus Dei zelf ?
Ik werd gelukkig nauwelijks gehinderd door enige diepgaande kennis van zaken .
Ik ben zeer verbaasd te horen dat deze graancirkel door de mens gemaakt is .
Dat is ons achteraf door de Zuid-Afrikanen in dank afgenomen , aldus Bruno Valkeniers .
Maar die werd door sommige ouders juist verweten de beweging te willen recupereren .
vooral daar word ik wel door uitgedaagd
juist hier word ik wel door uitgedaagd
nog overal word ik door uitgedaagd
Het huis is binnengedrongen
Het huis is binnengedrongen door de soldaten
Het huis is genaderd
In 1418 werd het verwoest door de Utrechters en door bendes uit Gelderland .
Je wordt door de hond of door de kat gebeten ...
vooral door hard ingrijpen werd hij gered
Hij sticht de nederzetting die later naar hem genoemd zal worden
Een kleine strook land van de oude gemeente Wognum zal worden overgeheveld naar de gemeente Hoorn .
De volgende beslissing zal worden genomen op 9 november 1939
De huidige 8000 m² zal worden uitgebreid tot bijna 13.000 m²
Non-theïstische religies dienen overigens niet verward te worden met het atheïsme .
het metaal dient zorgvuldig te worden behandeld
Deze overdracht van macht dient nauwkeurig bewaakt te worden
Het zelfde dient gezegd te worden van de prediking in 1206 door Dominicus Guzman
De regel dat de toestemming eerst geverifieerd dient te worden .
In dat geval hoefden de wiskundevakken niet gevolgd te worden
Er bestaat geen consensus onder wichelroedelopers over hoe het instrument gebruikt dient te worden .
Een WSDL-document hoeft niet met de hand geschreven te worden .
In dat geval hoeft de nalatenschap vervolgens niet geaccepteerd te worden .
Vervolgens dienen deze gegevens onder de aandacht te worden gebracht van personen en instanties
Bij onderzoek bleek er geen oorsmeer verwijderd te worden door oorkaarsen .
Door de positie van de baby dient extra aandacht gegeven te worden aan het functioneren van de ontlasting .
Leviticus 24:16 bepaalt dat het lasteren van Gods Naam moet worden gestraft met de dood door steniging .
De zoetstof aspartaam is bijvoorbeeld een bekende bron van fenylalanine die vermeden moet worden .
Soms bestaat de digitale oscilloscoop uit een kastje wat met de computer verbonden moet worden .
het betekent dat « gebiedsvreemd water » moet worden ingelaten .
Het gevolg is dat de septic tank met enige regelmaat leeggemaakt moet worden .
Of Nessie bestaat zal waarschijnlijk nooit bewezen worden .
Opgemerkt moet worden dat deze indeling niet compleet zal zijn
Bijna iedere sonate heeft een tweedelige structuur die in beginsel moet worden verdubbeld .
