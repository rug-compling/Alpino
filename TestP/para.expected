De buurvrouw heeft me naar huis gebracht .
De buurvrouw had mij naar huis gebracht .
De buurvrouw brengt me naar huis .
De buurvrouw bracht me naar huis .
De buurvrouw verzorgde de kinderen die de buurvrouw verzorgde .
De buurvrouw heeft de kinderen die de buurvrouw verzorgd heeft verzorgd .
Wie had toen de buurvrouw verzorgd ?
Wie verzorgde toen de buurvrouw ?
Een bijzonderheid : het eigen personeel van Fina voerde de enquête uit .
Professionele gebruikers testen weer gedurende een langere periode een tweehonderdtal exemplaren van het nieuwe gereedschap .
Op vraag van grote gebouweigenaars en van de schoonmaakindustrie ontwikkelde het TNO in Delft dat systeem .
Onze meest ervaren ambulante objectleider start elke nieuwe werf op .
Museumconsulenten begeleiden musea die nog niet aan de eisen kunnen voldoen .
Diverse CEN-comités bestuderen momenteel de ondertussen ontwikkelde manuals .
Quality Control test ook het eindproduct zelf , bijvoorbeeld een autonavigatie-cd .
In eerste instantie onthalen gebruikers warm deze meestal dure systemen .
De Papi-methodiek haalt die uit .
Slechts enkele bedrijven in België passen nochtans vandaag de dag de oplossing hiervoor toe nl. een gezond verliesbeheersingsbeleid .
Dit compenseert dan weer de uitstraling van deze Lancia .
Het Brits-Duitse Rover ( BMW nam Rover over ) is een apart merk .
Doordringen op een fleetmarkt die de Europese merken beheersen was voor Mitsubishi niet evident .
Denk maar aan de omniumverzekering op voertuigen die grote gespecialiseerde maatschappijen leasen .
De concessiehouders brengen zo de wagens in de handel .
Deze specificaties sluiten merken die misschien via een andere techniek eenzelfde doelstelling bereiken dikwijls uit .
Want de aanwezigheid van een justitie-antenne stigmatiseert meteen een wijk .
Dat vroeg me toch het kabinet .
Aan de Albertsquare heeft de gemeente in een twijfelachtig kunstwerk de samenlevingsproblemen gegoten .
Belgen bewonen drievierde van de flats - een uitzondering in deze omgeving .
Franky Van der Elst heeft het Belgische voetbal van een gewisse dood gered .
Metz had al eerder kansloos Moeskroen weggeveegd in de Uefabeker .
Drie uitstekende stukken redden toch deze bundel .
Uitgebreid onderzoek bevestigde dat .
Wereldwijd beoefenen meer dan drie miljoen geregistreerde spelers Magic .
Het outplacementbureau New Challenge Company vangt die na hun ontslag op .
Zijn oom Proxenus voedde hem op .
Opdrachtgevers zien dit vaak als keurmerk .
In Andorra vertegenwoordigt een afgevaardigde hen .
Gevaren bedreigen sommige gebieden met een menselijke oorzaak .
In 2003 trof een zware aardbeving het noorden .
De president benoemt de eerste minister .
Bij het ontwerpen van de programmeertaal Perl inspireerde AWK Larry Wall .
Volgens sommige auteurs hebben mensen aids gemaakt .
Ziekten plaagden haar in haar verdere leven .
De Nederlanders namen in 1636 Aruba over .
Vastenavond en het carnaval gaat Aswoensdag vooraf .
De armoedegrens duidt het verschil tussen armoede en welstand aan .
Reynier Adriaensen heeft het amok maken in Maleisië eind 17e eeuw beschreven .
Incourante afstanden waarvoor de IAAF records bijhoudt zijn de volgende .
Hitler heeft het antisemitisme niet uitgevonden .
De minister van Defensie en Ontwikkelingssamenwerking van Luxemburg had hem vergezeld .
Veel geld vloeide er naar de speciale regelingen die de regering had opgelegd .
Maar Karel De Gucht heeft de premier opgejut .
Hij weet dat zeer velen hem haten .
De beursgigant slokte genadeloos nutsbedrijven oude stijl op .
Sindsdien vergezelden bodyguards hem altijd .
Mensen bedienen immers computers .
De pensioenfondsen maken meer en meer de markt .
De meeste fondsen hielden de negatieve koersevolutie beperkter .
Op haar beurt heeft een markttrend die verandering veroorzaakt .
IBM Global Network verzorgt trouwens binnenshuis het beheer van het netwerk .
Vooral op lange termijn wint een goede voorbereiding veel tijd .
Velen beschouwen dit remsysteem als het veiligste ter wereld .
Een Japanse groep had ondertussen het bedrijf overgenomen .
Een Westvlaamse kartonfabriek die wil investeren in Uganda heeft mij gevraagd .
Mazda-importeur Beherman nam overigens de bouw van de stand zelf ter hand .
Waarom moest Aziza voor neonazi's onderduiken en belaagden neofascisten haar nicht ?
Diverse genetische fouten veroorzaken de ziekte van Alzheimer .
In 1987 kaapten de Zwitsers onze nationale chocolade-olifant .
Het toerisme werkt het plunderen hier blijkbaar in de hand .
Wij wensen niets liever dan dat onbevooroordeelde specialisten ons werk beoordelen .
Zo wandel ik op straat en herkent niemand mij .
Seks , geweld en hebzucht drijft iedereen .
Rokers verpesten gezonde lucht .
Uw cijfers wekken wel die suggestie .
De Japanse bezetter wantrouwde de ambonees .
Media pakken alleen wat zich gemakkelijk aandient op .
Tony ' Scarface ' Mantana maakt de surfer voor fucking cockroach uit .
Deze gedachte heeft lange tijd ook de geschiedschrijving in Latijns Amerika beïnvloed .
Planten maken gifstoffen om te voorkomen dat dieren ze opeten .
Hevige regens hebben de modderstroom veroorzaakt .
Gewone mensen plegen zelden bomaanslagen .
Zo heeft men internationaal onderzocht welke vrouwen mannen attractief vinden .
Het insect levert dus naar alle waarschijnlijkheid die stoffen aan .
Dat daagt me wel uit .
Het daagt me wel uit .
Dit daagt me wel uit .
Alles daagt me uit .
Niets daagt me meer uit .
Iets daagt me uit .
Dit daagt me uit .
Dat daagt me uit .
Het daagt me uit .
Sommigen hebben toen geprotesteerd .
Sommigen protesteerden toen .
Aan sommige tafeltjes applaudisseren oudere dames .
Vrijwel alle spelers speelden nu eenmaal zwak .
Tientallen demonstranten herinnerden buiten daaraan .
En daar kopen sommigen meer dan normaal .
Pro-Russische betogers hebben vannacht heftig geprotesteerd tegen die beslissing .
Buiten het gerechtshof betoogden slachtoffers van de apartheid .
Niemand twijfelt er aan wie de winnaar van de verkiezingen zal zijn .
Ook in Italië oefenen druk grondtroepen .
Het Nationaal Influenza-centrum in Rotterdam onderzoekt welke virus het nu precies is .
Hulporganisaties bereiken nauwelijks enkele duizenden van hen .
De linkse vakbond had opgeroepen tot de betoging .
Al heeft de aanstelling van de nieuwe partijleider Faymann waarschijnlijk erger voorkomen .
Een peperduur toestel waar dag en nacht leerling-piloten op oefenen .
In Iran hebben de tegenstanders van president Ahmadinejad opnieuw massaal betoogd .
Wanneer Boonen opnieuw zal koersen deelt Quick-Step ten gepaste tijde mee .
Over de crisis overlegt straks nog het Midden-Oosten kwartet .
De familie heeft opgeroepen tot de optocht .
De Verenigde Staten heeft daar vermoedelijk achter de schermen voor geijverd .
Het parlement had daar gisteren al toe opgeroepen .
Het is niet de eerste keer dat het Franse overheidspersoneel staakt .
De aandeelhouders van Fortis dringen al een tijdje op zijn aftreden aan .
Daarom hamsteren de Spanjaarden al massaal .
Ook nu schoot de politie .
De electriciteitsproducenten hebben nog niet bevestigd dat er een akkoord is .
Een lokale militie van Zuid-Ossetië heeft mogelijk gevuurd .
De linkse vakbond CGTP had opgeroepen tot de betoging .
De Arabische geleerde Averroes heeft veel van zijn werken vertaald .
Ik had de schat gevonden maar een enorm zeemonster bewaakte hem .
Te veel dingen tegelijk teisterden me .
De gordijnen waren open en het vroege ochtendlicht omlijstte haar hologige gezicht .
Een schorre vloek en een plons verstoorde plotseling ruw die stilte .
Twee motorrijders beslopen toen pa hem volgde hem van achteren .
Een bekraste keukentafel nam het merendeel van de ruimte in beslag .
Ik geloof dat mijn moeder je hielp .
Het soort boosheid dat zich voordoet als verbazing greep hem .
Het begint erop te lijken dat een buitenstaander de brand heeft gesticht .
Steunde Opus Dei zelf die immers niet ?
Enige diepgaande kennis van zaken hinderde me gelukkig nauwelijks .
Ik ben zeer verbaasd te horen dat de mens deze graancirkel heeft gemaakt .
De Zuid-Afrikanen hebben ons dat achteraf in dank afgenomen , aldus Bruno Valkeniers .
Maar sommige ouders verweten die juist de beweging te willen recupereren .
Vooral dat daagt me wel uit .
Juist dit daagt me wel uit .
Nog alles daagt me uit .
Men is het huis binnengedrongen .
De soldaten zijn het huis binnengedrongen .
Men is het huis genaderd .
In 1418 verwoestte de Utrechters en bendes uit Gelderland het .
De hond of de kat bijt je .
Vooral hard ingrijpen redde hem .
Hij sticht de nederzetting die men later naar hem zal noemen .
Men zal een kleine strook land van de oude gemeente Wognum naar de gemeente Hoorn overhevelen .
Op 9 november 1939 zal men de volgende beslissing nemen .
Men zal de huidige 8000 m² tot bijna 13.000 m² uitbreiden .
Overigens dient men Non-theïstische religies niet te verwarren met het atheïsme .
Het metaal dient men zorgvuldig te behandelen .
Nauwkeurig dient men deze overdracht van macht te bewaken .
Men dient het zelfde van de prediking in 1206 door Dominicus Guzman te zeggen .
De regel dat men eerst de toestemming dient te verifiëren .
In dat geval hoefde men de wiskundevakken niet te volgen .
Er bestaat geen consensus onder wichelroedelopers over hoe men het instrument dient te gebruiken .
Een WSDL-document hoeft men niet met de hand te schrijven .
In dat geval hoeft men vervolgens de nalatenschap niet te accepteren .
Vervolgens dient men deze gegevens onder de aandacht van personen en instanties te brengen .
Bij onderzoek bleken oorkaarsen geen oorsmeer te verwijderen .
De positie van de baby dient extra aandacht aan het functioneren van de ontlasting te geven .
Leviticus 24:16 bepaalt dat men met de dood door steniging het lasteren van Gods naam moet straffen .
De zoetstof aspartaam is bijvoorbeeld een bekende bron van fenylalanine die men moet vermijden .
De digitale oscilloscoop bestaat soms uit een kastje wat men met de computer moet verbinden .
Het betekent dat men gebiedsvreemd water moet inlaten .
Het gevolg is dat men de septic tank met enige regelmaat moet leegmaken .
Waarschijnlijk zal men nooit bewijzen of Nessie bestaat .
Men moet opmerken dat deze indeling niet compleet zal zijn .
Bijna iedere sonate heeft een tweedelige structuur die men in beginsel moet verdubbelen .
